README.md

```diff
@@ -89,7 +89,7 @@ import (
 func main() {
 	client := chroma.NewClient("http://localhost:8000")
 	collectionName := "test-collection"
-	metadata := map[string]interface{}{}
+	Metadata := map[string]interface{}{}
```

Comment:
@tazarov Generally we will use lowercase at the beginning of the variable name

`Metadata` -> `metadata`

Reply:
I've update the whole example in a stacked PR #38